src/ringmesh/basic/geometry.cpp

```diff
@@ -48,6 +48,9 @@
 
 namespace RINGMesh
 {
+    const std::array< std::array< index_t, 3 >, 4 > Geometry::Tetra::tetra_facet_vertex = { {
```

Comment:
Why these changes here ?

Reply:
Not related to cmake but I had some issues at some point during the development. This was a way to resolve it. I leave the code because I think this is better than using an obscure geogram struct.